modules/renter/hostdb/hostdb.go

```diff
@@ -58,5 +118,7 @@ func New() *HostDB {
 	}
 	go hdb.threadedScan()
 
-	return hdb
+	cs.ConsensusSetSubscribe(hdb)
```

Comment:
?

huh. I guess it doesn't have any automatic actions that it needs to participate in.